examples/tree_1d_dgsem/elixir_linearizedeuler_gauss_wall.jl

```diff
@@ -0,0 +1,63 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the linearized Euler equations
+
+equations = LinearizedEulerEquations1D(v_mean_global = 0.5, c_mean_global = 1.0,
+                                       rho_mean_global = 1.0)
+
+solver = DGSEM(polydeg = 5, surface_flux = flux_hll)
+
+coordinates_min = (0.0,)
+coordinates_max = (90.0,)
+
+# Create a uniformly refined mesh with periodic boundaries
+mesh = TreeMesh(coordinates_min, coordinates_max,
+                initial_refinement_level = 6,
+                n_cells_max = 100_000,
+                periodicity = false)
+
+function initial_condition_gauss_wall(x, t, equations::LinearizedEulerEquations1D)
```

Comment:
Maybe add a comment as to what type of solution this initial condition gives. It is basically a Gaussian pulse that splits into left-going and right-going waves and then bounce off the walls, right?